tests/test_htm.py

```diff
@@ -33,22 +36,42 @@ def forward(self, action, obs):
         return self.q(x)
 
 
-class Policy(nn.Module):
-    def __init__(self, obs, act):
-        super().__init__()
-        self.pi = nn.Sequential(
-            nn.Linear(obs, N_HIDDEN),
-            nn.ReLU(),
-            nn.Linear(N_HIDDEN, N_HIDDEN),
-            nn.ReLU(),
-            GaussianPolicyHead(N_HIDDEN, act),
-        )
+def test_htm():
+    device = torch.device("cpu")
+    env = DictGymWrapper(AsyncVectorEnv(10 * [HitTheMiddle]))
+    table = DictObsTable(spaces=env.dict_space, maxlen=1000, device=device)
+    memory_proxy = TableMemoryProxy(table)
+    dataloader = MemoryLoader(table, 100, 2, "batch_size")
+
+    q1 = QNet(2, 1)
+    q2 = QNet(2, 1)
+    policy = Policy(2, 1, [N_HIDDEN, N_HIDDEN])
```

Comment:
I think I was trying to get away from having the "smart" convenience class `GaussianMLlpPolicy`. I think it's a mistake to have those kinds of convenience wrappers, it is much better to make constructing a policy very easy. But but, it's still there. I can refactor it later.

Reply:
Yepp, let's think about that later.